smoderp2d/exceptions.py

```diff
@@ -48,14 +48,31 @@ class ConfigError(Exception):
     pass
 
 
+class SmallParameterValueError(ConfigError):
+    """ Exception raised if a parameter reaches a wrong numeric value
+    """
+    def __init__(self,param, value, limit):
+        self.msg = "Parameter '{}' has low value ({} < {}).".format(
+                param, value, limit
+        )
+        Logger.fatal(self.msg)
+
+
+class LargeParameterValueError(ConfigError):
+    """ Exception raised if a parameter reaches a wrong numeric value
+    """
+    def __init__(self,param, value, limit):
+        self.msg = "Parameter '{}' has a wrong value ({} > {}).".format(
```

Comment:
wrong or large?

Reply:
"{} parameter value is too large ({} > {})"